Documentation/ApiOverview/SessionStorageFramework/UserSessionManagement.rst

```diff
@@ -97,10 +97,10 @@ The session object created or retrieved by the :php:`UserSessionManager` provide
 +---------------------+-------------+------------------------------------------------------------------------------+
 | Method              | Return type | Description                                                                  |
 +=====================+=============+==============================================================================+
-| getIdentifier()     | String      | Returns the session id. This is the :php:`ses_id` respectively the           |
```

Comment:
I would prefer to have this information in the source code and generate the API information automatically so it stays updated...

Reply:
Yup, I agree and thought also about it, but it is not part of this PR.